grpc-bridge-runtime/src/main/java/org/jboss/resteasy/grpc/runtime/servlet/GrpcHttpServletDispatcher.java

```diff
@@ -0,0 +1,58 @@
+package org.jboss.resteasy.grpc.runtime.servlet;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.jboss.resteasy.plugins.server.servlet.HttpServlet30Dispatcher;
+
+import jakarta.servlet.Servlet;
+import jakarta.servlet.ServletConfig;
+import jakarta.servlet.ServletContext;
+import jakarta.servlet.ServletException;
+
+public class GrpcHttpServletDispatcher extends HttpServlet30Dispatcher {
+
+   private static final long serialVersionUID = -7323100224345687064L;
+   private static final Map<String, Servlet> servletMap = new HashMap<String, Servlet>();
+   private static final Map<Servlet, ServletContext> servletContextMap = new HashMap<Servlet, ServletContext>();
```

Comment:
We probably need to think about this and how it works in environments where this would be shared across multiple deployments. The servlet names could be the same which could cause some issues.